ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/host1.cfg

```
@@ -3547,6 +3558,1102 @@ router bfd
       reflector min-rx 600
       reflector local-discriminator 155.1.3.1
 !
+router bgp 65101
+   bgp asn notation asdot
+   router-id 192.168.255.3
+   update wait-for-convergence
+   update wait-install
+   bgp default ipv4-unicast
+   bgp default ipv4-unicast transport ipv6
+   distance bgp 20 200 200
+   graceful-restart restart-time 555
+   graceful-restart stalepath-time 666
+   graceful-restart
+   graceful-restart-helper restart-time 888
+   bgp route-reflector preserve-attributes always
+   maximum-paths 32 ecmp 32
+   no bgp default ipv4-unicast
```

First and last lines of this block are contradicting
```suggestion
   bgp default ipv4-unicast
   bgp default ipv4-unicast transport ipv6
   distance bgp 20 200 200
   graceful-restart restart-time 555
   graceful-restart stalepath-time 666
   graceful-restart
   graceful-restart-helper restart-time 888
   bgp route-reflector preserve-attributes always
   maximum-paths 32 ecmp 32
```

---

The last line probably comes from `bgp_defaults` which is similar to raw_eos_cli

---

yes so i will move it in different hosts